src/vma/proto/route_val.cpp

```diff
@@ -131,3 +134,12 @@ void route_val::print_val()
 	set_str();
 	rt_val_logdbg("%s", to_str());
 }
+
+void route_val::set_mtu(uint32_t mtu)
+{
+	if (mtu > g_p_net_device_table_mgr->get_max_mtu()) {
+		rt_val_logdbg("route mtu cannot be bigger then max mtu set on devices");
+	} else {
+		m_mtu = mtu;
+	}
+}
```

Comment:
please consider using inline function as
```
inline void set_mtu(uint32_t mtu) {m_mtu = (mtu > g_p_net_device_table_mgr->get_max_mtu() ? g_p_net_device_table_mgr->get_max_mtu() : mtu)}
```

Reply:
its a problem to put it in the h due to compilation issue not sure it's worth it

Reply:
got it